monitoring/heartbeats/in-servicepulse.md

```diff
@@ -6,38 +6,62 @@ redirects:
 
 > [!NOTE]
-> This is a configuration setting only and the endpoint itself may still be configured to send heartbeat messages.
+> This feature requires ServicePulse version 1.42.0 or higher and ServiceControl version 5.9.0 or higher.
+
+### Track Instances
+
+`Track Instances` is the best setting for endpoints where all instances are hosted in physical infrastructure that is not auto-scaled. For example, physical or virtual servers. All current and historical instances will be tracked, unless the instance is [deleted](#instances-removing-endpoint-instances). If any instances are not receiving heartbeats, or if any instances have been muted, then the endpoint is marked as unhealthy.
+
+### Do Not Track Instances
+
+`Do Not Track Instances` is the best setting for endpoints that are hosted in infrastructure with autoscalers. For example, Kubernetes, Azure Container Apps and AWS Elastic Container Service. Historical instances are not tracked, and only currently alive instances will be shown. The endpoint will be marked as unhealthy if there are no alive instances, and will be automatically removed after 6 hours of inactivity.
+
+## Instances
 
-## Removing endpoints
+Each logical endpoint has one or many physical or virtual instances that perform the actual work of handling or sending messages.
 
-When an endpoint has been decommissioned and should no longer be displayed in ServicePulse it can be removed. From the Endpoint Heartbeats screen, click the trash bin icon next to an endpoint's name to remove it from ServicePulse. The trash icon is only visible when displaying endpoint instances. It is not shown when displaying logical endpoints.
+![Instances](sp-endpoint-instances.png "width=800")
+
+### Muting Instances
+
+By default, each new endpoint instance discovered by ServicePulse is monitored for heartbeats. If the instance is not sending heartbeats (either because it is offline or because it does not have the heartbeats plugin installed) it will immediately be marked as unhealthy and an alert will show in the navigation and on the dashboard. If this is not desired, E.g. if the instance is intentionally offline for maintenance or redeployment, then the instance can be muted which will prevent alerts from being shown. The endpoint will still be marked as unhealthy, as a reminder to unmute once maintenance is complete.
+
+There are bulk operation buttons at the top of the screen to allow for muting or unmuting all instances of an endpoint at once.
+
+### Removing Endpoint Instances
+
+When an endpoint instance has been decommissioned, and should no longer be displayed in ServicePulse, it can be removed. 
+- For [Tracked Instances](#configuration-track-instances), click the `Delete` button on the instance row to remove it from ServicePulse
+- For [Untracked Instances](#configuration-do-not-track-instances) the endpoint will automatically be removed after 6 hours of inactivity.
```

Comment:
```suggestion
- For [Untracked Instances](#configuration-do-not-track-instances), the endpoint instance will automatically be removed after 6 hours of inactivity.
```

Reply:
as far as the user is aware, there _are_ no instances, so it's the endpoint being removed not the instance

Reply:
it is the instances that are being auto-pruned, not the logical endpoint